meta/main.yml

```diff
@@ -9,13 +9,16 @@ dependencies:
       - build
 
   - role: sansible.users_and_groups
-    version: v1.0
+    version: v1.4.1
     users_and_groups:
       groups:
         - name: "{{ logstash.group }}"
       users:
         - name: "{{ logstash.user }}"
           group: "{{ logstash.group }}"
+      sudoers: []
+      whitelist_groups: []
+      authorized_keys_dir:
```

Comment:
I don't think you need to set sudoers, whitelist_groups or authorized_keys_dir, they are already set to sensible defaults in sansible.users_and_groups.

Reply:
With Ansible 2.2+, I get an error if I don't define these values, even with `hash_behavior = merge`.